src/content/migrate/5.md

```diff
@@ -67,6 +67,7 @@ Update the following options to their new version (if used):
 - `NamedChunksPlugin` => `optimization.chunkIds: 'named'`
 - `HashedModulesPlugin` => `optimization.moduleIds: 'hashed'`
 - `optimization.occurrenceOrder: true` => `optimization: { chunkIds: 'total-size', moduleIds: 'size' }`
+- Rename `optimization.splitChunks.cacheGroups.vendors` => `optimization.splitChunks.cacheGroups.defaultVendors`
```

Comment:
```suggestion
- Renamed `optimization.splitChunks.cacheGroups.vendors` => `optimization.splitChunks.cacheGroups.defaultVendors`
```

Reply:
Thanks for the really quick review :), Made apt changes